inbox/react/components.mdx

```diff
@@ -19,53 +19,50 @@ It renders an opinionated `<Inbox />` with a bell button, that triggers a popove
 
 #### Default Inbox
 
-```jsx
-import { Inbox } from "@novu/react";
+```tsx
+import { Inbox } from '@novu/react';
 
 function Novu() {
   return (
     <Inbox
-      options={{
-        subscriberId: "SUBSCRIBER_ID",
-        applicationIdentifier: "APPLICATION_IDENTIFIER",
-      }}
+      applicationIdentifier="YOUR_APPLICATION_IDENTIFIER"
+      subscriberId="YOUR_SUBSCRIBER_ID"
```

Comment:
Here we have mainly improvements on the props, however some sections were duplicated with a separate page.